src/Runtime.Base/src/System/Runtime/ThunkPool.cs

```diff
@@ -293,12 +293,12 @@ private IntPtr TryGetThunkDataAddress(IntPtr thunkAddress)
                 return IntPtr.Zero;
 
             // Compute the thunk's index
-            int thunkIndex = (int)((thunkAddressValue - currentThunksBlockAddress) / (nuint)Constants.ThunkCodeSize);
+            nuint thunkIndex = (thunkAddressValue - currentThunksBlockAddress) / (nuint)Constants.ThunkCodeSize;
```

Comment:
`thunkIndex` is int in other places in the file. If you would like to change this, change it everywhere.